public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx

```diff
@@ -136,6 +136,9 @@ export const QueryWrapper = ({
   }
 
   const showVizualisation = data.state !== LoadingState.NotStarted;
+  // ⚠️ the query editors want the entire array of queries passed as "DataQuery" NOT "AlertQuery"
+  // TypeScript isn't complaining here because the interfaces just happen to be compatible
+  const editorQueries = cloneDeep(queries.map((query) => query.model));
```

Comment:
Leaving this here for posterity; some editors mutate the `queries` prop they receive and those are managed by an immutable reducer on the Alerting side – so we clone them before passing them forward.